cmd/kind/kind.go

```diff
@@ -20,40 +20,76 @@ package kind
 import (
 	"os"
 
+	"github.com/sirupsen/logrus"
+
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 
 	"sigs.k8s.io/kind/cmd/kind/build"
 	"sigs.k8s.io/kind/cmd/kind/create"
 	"sigs.k8s.io/kind/cmd/kind/delete"
+	logutil "sigs.k8s.io/kind/pkg/log"
 )
 
+const defaultLevel = logrus.WarnLevel
+
+// Flags for the kind command
+type Flags struct {
+	LogLevel string
+}
+
 // NewCommand returns a new cobra.Command implementing the root command for kind
 func NewCommand() *cobra.Command {
+	flags := &Flags{}
 	cmd := &cobra.Command{
 		Use:   "kind",
 		Short: "kind is a tool for managing local Kubernetes clusters",
 		Long:  "kind creates and manages local Kubernetes clusters using Docker container 'nodes'",
+		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+			return runE(flags, cmd, args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
 	}
+	cmd.PersistentFlags().StringVar(&flags.LogLevel, "loglevel", defaultLevel.String(), "logrus log level")
```

Comment:
Can we add `e.g. info, error, warn` or something to this? It gets quite frustrating having to remember what strings to use for all different log levels 😅

Reply:
Yeah, this flag info is not particularly helpful.

Reply:
added this